src/use-format-block.hook.js

```diff
@@ -6,10 +6,44 @@ export function useFormatBlock() {
   return {
     formatBlock(value) {
       if (value === document.queryCommandValue('formatBlock')) {
-        performCommandWithValue('div')
+        if (!!window.chrome && isInHTMLList()) {
+          removeHeader()
```

Comment:
The `useFormatBlock` hook is not specific to headers. It also can be used for `<address>`, `<pre>`, `<div>`, and really any other block level element.

My concern is that this code here would make the hook only work for headers.

Reply:
agreed, updated to check if formatting header